Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -3404,6 +3404,9 @@ public void Actions_CanPerformContinuousHold()
     [Property("TimesliceEvents", "Off")]
     public void Actions_CanPerformHoldOnTrigger()
     {
+        var oldPressPoint = InputSystem.settings.defaultButtonPressPoint;
```

Comment:
This is actually not necessary. A new settings object is created for every test and then thrown away.